Automattic/jetpack
# Subscription block: don't include Publicize connections in subscriber count

When `Show subscriber count` is toggled on for the subscription block, a count is shown that includes all followers on all enabled Publicize connections (Twitter followers, LinkedIn followers, ...). This confuses many users.

This PR fixes that by including an extra parameter in the XML-RPC call, which when enabled only fetches the subscribers.

The patch for the XML-RPC server side can be found here D89641-code


Fixes #25012 #23727 & https://github.com/Automattic/wp-calypso/issues/66213

#### Changes proposed in this Pull Request:
* Exclude Publicize followers in the subscription block follower count

#### Does this pull request change what data or activity we track or use?
No

#### Jetpack product discussion
There has been some discussion about adding a toggle for this, but we settled on only showing the subscribers for now.  p1665388720958159/1664774419.931139-slack-C02NQ4HMJKV

#### Testing instructions:
1. Apply this PR
2. Enable Jetpack Social and connect to Twitter (Jetpack > Social > Manage Connections)
3. Add one e-mail follower to your Jetpack site + click the link in the confirmation mail
4. Create a new page/post and add the subscription block, make sure Show subscriber count is turned on in the block settings
